besu/src/main/java/org/hyperledger/besu/cli/subcommands/PublicKeySubCommand.java

```diff
@@ -113,22 +109,23 @@ public void run() {
       checkNotNull(parentCommand);
       checkNotNull(parentCommand.parentCommand);
 
-      parentCommand.getKeyPair().ifPresent(this::outputPublicKey);
```

Comment:
Was this change because NPE were being encountered?

Reply:
you mean not using getKeyPair anymore? or the usage of Optional.ofNullable?